lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,559 @@
+---
+name: Parameters
+field_name_base_url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters
```

Comment:
nit: `http` -> `https`

Reply:
Applies throughout the file

Reply:
updated throughout the file https://github.com/cerner/fhir.cerner.com/pull/780/commits/c8a1c8272a99dbf6bd094d88ad599ff945126235#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R3